frontend/src/views/MigrationHistoryDetail.vue

```diff
@@ -300,7 +300,15 @@ export default defineComponent({
         instanceStore.fetchInstanceList();
       }
 
-      return migrationHistoryList.filter((mh) => mh.id <= migrationHistoryId);
+      // The returned migration history list has been ordered by `sequence` DESC.
```

Comment:
Seems other databases return the migration history list order by id desc.
Should this be specific for Spanner?